worker/cmd/copier/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+	"github.com/reearth/reearth-cms/worker/internal/adapter/http"
+	rmongo "github.com/reearth/reearth-cms/worker/internal/infrastructure/mongo"
+	"github.com/reearth/reearth-cms/worker/internal/usecase/interactor"
+	"github.com/reearth/reearth-cms/worker/internal/usecase/repo"
+	"github.com/reearth/reearthx/log"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.opentelemetry.io/contrib/instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo"
+)
+
+func main() {
+	log.Infof("reearth-cms/worker: copier has started")
+	ctx := context.Background()
+
+	if err := godotenv.Load(".env"); err != nil && !os.IsNotExist(err) {
+		log.Fatal("config: unable to load .env")
+	} else if err == nil {
+		log.Infof("config: .env loaded")
+	}
+
+	dbURI := mustGetEnv("REEARTH_CMS_WORKER_DB")
+	collection := mustGetEnv("REEARTH_CMS_COPIER_COLLECTION")
+	filter := mustGetEnv("REEARTH_CMS_COPIER_FILTER")
+	changes := mustGetEnv("REEARTH_CMS_COPIER_CHANGES")
```

Comment:
I think those params should come from os.Args not from env 🤔

Reply:
I am not sure, can you please check how decompress is implemented:
https://github.com/reearth/reearth-cms/blob/ce70e176f6409ac07b2ba3ba2e143ab77cfb8515/server/internal/infrastructure/gcp/taskrunner.go#L175-L179